### ISSUE TITLE
[Bug]: Dvas installation package v1.1.0 used by pip miss some folders

### ISSUE THREAD

**Issue by msommer24** [MeteoSwiss/dvas#298]
### Is there already an issue for this?

- [X] I found no existing issue for this bug.

### Describe the bug

The installation package of dvas v1.1.0 miss some folders and files, so that running dvas is not possible. I had a look into the related file `dvas-1.1.0-py3-none-any.whl` which I got using `pip download dvas`. It seems, that there are missing some folders, e.g. `dvas_demo`, `dvas\plots\mpl_styles\`, and maybe some more.

### Steps to reproduce

1. I tried to install dvas using documented command `pip install dvas`.
2. After installation of dvas with all related packages, I tried to test the dvas installation with `dvas_init_arena -h`.
3. `FileNotFoundError` error occures  (see log output for details)

### Relevant log output

```shell
D:\>dvas_init_arena -h
Traceback (most recent call last):
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\helper.py", line 411, in check_path
    out = Path(value).resolve(strict=True)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\pathlib.py", line 1240, in resolve
    s = self._flavour.realpath(self, strict=strict)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 722, in realpath
FileNotFoundError: [WinError 2] Das System kann die angegebene Datei nicht finden: 'C:\\Users\\msommer\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\dvas\\plots\\mpl_styles'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Scripts\dvas_init_arena.exe\__main__.py", line 4, in <module>
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas_recipes\__main__.py", line 18, in <module>
    from .hl_commands import init_arena, run_recipe, optimize
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas_recipes\hl_commands.py", line 24, in <module>
    from dvas.data.strategy.data import GDPProfile
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\data\strategy\data.py", line 21, in <module>
    from ...database.database import DatabaseManager, InfoManager
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\database\database.py", line 38, in <module>
    from ..config.config import Prm as ParameterCfg
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\config\config.py", line 31, in <module>
    from ..environ import path_var
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\environ.py", line 205, in <module>
    path_var = GlobalPathVariablesManager()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\helper.py", line 65, in __call__
    instance = super(SingleInstanceMetaClass, cls).__call__(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\environ.py", line 146, in __init__
    super().__init__()
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\environ.py", line 55, in __init__
    self.set_default_attr()
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\environ.py", line 85, in set_default_attr
    setattr(
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\helper.py", line 339, in __set__
    instance.__dict__[self._name] = self._setter_fct(
                                    ^^^^^^^^^^^^^^^^^
  File "C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\helper.py", line 413, in check_path
    raise TypeError(f"Path '{value}' does not exist") from first_error
TypeError: Path 'C:\Users\msommer\AppData\Local\Programs\Python\Python312\Lib\site-packages\dvas\plots\mpl_styles' does not exist
```

### Environment

```markdown
- OS: Windows 10
- Python: 3.12
- dvas: 1.1.0
```

### Code of Conduct

- [X] I agree to follow the dvas Code of Conduct.

**Comment by fpavogt** [comment]
Thanks for the report - I can reproduce the problem on my end. Will track it down. In the meantime, you can get all the necessary files from the [Github release page](https://github.com/MeteoSwiss/dvas/releases), and install the package manually.

However, because it has been a while, doing so will trigger further bugs. Since the code was released, numpy and other packages evolved in breaking ways. I'll need to cap the supported versions in the ``setup.py`` file. Stay tuned for updates.

**Comment by fpavogt** [comment]
The problem was caused by outdated paths in ``MANIFEST.in``, and will be corrected in #300. This had not been spotted before because of https://github.com/pypa/setuptools/issues/436. Thanks again for the report @msommer24 !
